notary-plugins/notary-plugin-non-validating/notary-plugin-non-validating-server/src/test/kotlin/com/r3/corda/notary/plugin/nonvalidating/server/NonValidatingNotaryServerFlowImplTest.kt

```diff
@@ -322,7 +329,12 @@ class NonValidatingNotaryServerFlowImplTest {
             on { size } doReturn 1
         }
 
-        // 2. Check if any filtered transaction data should be overwritten
+        // 2. Get current notary and parse its data
+        whenever(mockMemberLookup.myInfo()).thenReturn(notaryInfo)
+        whenever(mockMemberLookup.myInfo().memberProvidedContext).thenReturn(memberProvidedContext)
```

Comment:
Wouldn't it make sense to do the mocking on the objects that are defined already anyway? That way, adding or removing a temporary val won't randomly break a test.
```suggestion
        whenever(notaryInfo.memberProvidedContext).thenReturn(memberProvidedContext)
```

Reply:
Right, that makes sense to me.